activestate.generators.yaml

```diff
@@ -120,3 +120,22 @@ scripts:
       go run scripts/ci/payload-generator/main.go -b ${TEST_CHANNEL} -v ${TEST_VERSION}
       copy_test_payload
       go run scripts/ci/update-generator/main.go -b ${TEST_CHANNEL} -v ${TEST_VERSION} -o ${TEST_ARCHIVE_DIR}
+  - name: generate-versions-list
+    language: bash
+    standalone: true
+    description: Generates master versions.json from S3 and info.json's from generate-update
+    value: |
+      set -e
+      $constants.SET_ENV
+
+      go run scripts/ci/update-version-list/main.go ./build/update
+  - name: generate-test-versions-list
+    language: bash
+    description: Generates test versions.json
+    value: |
+      set -e
+      $constants.SET_ENV
+
+      $scripts.generate-test-update
+
+      go run scripts/ci/update-version-list/main.go ./build/test-update
```

Comment:
I think we can drop this considering it's not used, and if you want to test the script yourself you can just run it directly (ie. this isn't solving some big testing hurdle if I understand correctly).